src/frontend/state/ContextProvider.tsx

```diff
@@ -31,7 +31,7 @@ const initialContext: ContextType = {
   libraryStatus: [],
   libraryTopSection: 'disabled',
   handleLibraryTopSection: () => null,
-  platform: 'unknown',
+  platform: 'linux',
```

Comment:
set this as default since it is the majority that uses Heroic. It was causing a unnecessary re-render since it was changing from `unknown`.
Will still cause this re-render on other OSes on the first load, but then the value will be stored on browser cache and loaded from there.